tools/@aws-cdk/node-bundle/src/notice.ts

```diff
@@ -0,0 +1,264 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import type { Dependency, Attribution, NoticeViolations } from './model';
+import { shell } from './shell';
+
+/**
+ * Valid licenses that are ok to redistribute.
+ */
+const DEFAULT_VALID_LICENSES = [
+  'Apache-2.0',
+  'MIT',
+  'BSD-3-Clause',
+  'ISC',
+  'BSD-2-Clause',
+  '0BSD',
+];
+
+/**
+ * NOTICE file path.
+ */
+const FILE_PATH = 'NOTICE';
+
+/**
+ * String signifying the attributions segement is to follow.
+ */
+const ATTRIBUTIONS_START = 'This package includes the following third-party software:';
+
+/**
+ * ---------------
+ */
+const ATTRIBUTIONS_SEPARATOR = `\n${'-'.repeat(15)}\n`;
```

Comment:
I consider it's bad form to runtime compute a string you can statically determine :)

```suggestion
const ATTRIBUTIONS_SEPARATOR = '\n---------------\n';
```

Reply:
Fair enough.